src/search.cpp

```diff
@@ -292,20 +292,16 @@ void Search::Worker::iterative_deepening() {
     Stack  stack[MAX_PLY + 10] = {};
     Stack* ss                  = stack + 7;
 
-    for (int i = 7; i > 0; --i)
```

Comment:
id prefer this to be unchanged.. it makes more logical that sense that to fill the entries in
-7
-6
-5
-4
-3
-2
-1
0
1

order than

-1
-2
-3
-4
-5
-6
-7
0
1

Reply:
Ok, reverted